### ISSUE TITLE
why is it generating extra column in markdown tables?

### ISSUE THREAD

**Issue by agauniyal** [neogeek/doxdox#52]
I see extra column being generated in markdown tables filled with `| &nbsp; |`. What is the reasoning behind this and can this be disabled?

**Comment by neogeek** [comment]
Is it the final column? This is for the optional tag. I was looking into removing it completely if nothing was optional. Are you seeing a completely empty column?

Example: https://github.com/neogeek/raspar/blob/master/DOCUMENTATION.md

**Comment by agauniyal** [comment]
@neogeek yes, instead of `Optional` I've `&nbsp;` in the last column. You can see it here too - https://github.com/neogeek/doxdox/blob/master/DOCUMENTATION.md
